platform/mellanox/docker-syncd-mlnx/lib/port_utils.py

```diff
@@ -39,6 +37,30 @@ def get_ports_lanes_map(config_db):
 
     return lanes_map
 
+def get_port_max_width(handle):
+    """ Get max number of lanes in port according to chip type
+
+    Args:
+        handle (sx_api_handle_t): SDK handle
+
+    Returns:
+        int: max lanes in port
+    """
+    # Get chip type
+    chip_type = sx_get_chip_type(handle)
```

Comment:
What if this API (or those arrive after it) will return fail status ?

Reply:
When sx_get_chip_type will fail exception will be raised and handled by main.
I added missing RC check to rm_chip_limits_get.